src/main/java/Backend/FileDownloader.java

```diff
@@ -284,6 +286,14 @@ public void run() {
                             messageBroker.sendMessage(USER_INTERRUPTION, LOGGER_ERROR, "download");
                         } catch (IOException io1) {
                             messageBroker.sendMessage(FAILED_TO_DOWNLOAD_YOUTUBE_VIDEO, LOGGER_ERROR, "download");
+                            String message = e.getMessage();
```

Comment:
Handles the problem mentioned as an example in the issue of not having additional details about why the youtube download failed.